plugins/org.jboss.tools.openshift.core/META-INF/MANIFEST.MF

```diff
@@ -5,13 +5,19 @@ Bundle-SymbolicName: org.jboss.tools.openshift.core
 Bundle-Version: 3.0.0.qualifier
 Bundle-Activator: org.jboss.tools.openshift.internal.core.OpenShiftCoreActivator
 Bundle-Vendor: %Bundle-Vendor
-Require-Bundle: org.jboss.tools.openshift.common.core;bundle-version="[3.0.0,4.0.0)",
- org.jboss.tools.openshift.client;bundle-version="[3.0.0,4.0.0)",
+Require-Bundle: org.jboss.tools.openshift.client;bundle-version="[3.0.0,4.0.0)",
  org.jboss.tools.common.core;bundle-version="3.6.0",
- org.eclipse.core.runtime
+ org.eclipse.core.runtime,
+ org.jboss.tools.openshift.common.core;bundle-version="3.0.0",
+ org.apache.httpcomponents.httpclient;bundle-version="4.3.6",
+ org.apache.httpcomponents.httpcore;bundle-version="4.3.3",
+ org.apache.commons.codec;bundle-version="1.6.0",
+ org.jboss.ide.eclipse.as.dmr;bundle-version="3.1.0",
```

Comment:
@jcantrill I now spotted another incosistency: ConnectionRegistry is using ConnectionURL for all it's connections (v3 and v2). While we currently use ConnectionURL to serialize v2 connections we use json for v3. IMHO we should either use ConnectionURL for all connections when serializing or move ConnectionURL out of the registry.